installer/resources/pacbot_app/files/DB_Policy.sql

```diff
@@ -3592,10 +3606,12 @@ INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `def
 
 -- GCP VM --
 INSERT  IGNORE INTO cf_PolicyTable (policyId, policyUUID, policyName, policyDisplayName, policyDesc, resolution, resolutionUrl, targetType, assetGroup, alexaKeyword, policyParams, policyFrequency, policyExecutable, policyRestUrl, policyType, policyArn, severity, category, autoFixAvailable, autoFixEnabled, allowList, waitingTime, maxEmailNotification, templateName, templateColumns, fixType, warningMailSubject, fixMailSubject, warningMessage, fixMessage, violationMessage, elapsedTime, userId, createdDate, modifiedDate, status, enricherSource)
-VALUES ('gcp_vm_r7_resource_scanned','gcp_vm_r7_resource_scanned','GCPVMInstanceScannedByRapid7','Deploy Rapid7 agent in Virtual Machine instance','The Rapid7 Agent collect and analyz endpoint data, crucial for effective vulnerability management, threat detection, and IT operations. Its integration with Rapid7 suite of products enables organizations to maintain a robust security posture across their entire environment','','https://paladincloud.io/docs/vuln-policies','vminstance','gcp','GCPVMInstanceScannedByRapid7',
+VALUES ('gcp_vm_r7_resource_scanned','gcp_vm_r7_resource_scanned','GCPVMInstanceScannedByRapid7','Enable Rapid7 Vulnerability Scan','The Rapid7 Agent collect and analyz endpoint data, crucial for effective vulnerability management, threat detection, and IT operations. Its integration with Rapid7 suite of products enables organizations to maintain a robust security posture across their entire environment','','https://paladincloud.io/docs/vuln-policies','vminstance','gcp','GCPVMInstanceScannedByRapid7',
 '{"params":[{"encrypt":false,"value":"critical","key":"severity"},{"encrypt":false,"value":"security","key":"policyCategory"}]}','0 0 ? * MON *','','','ManagePolicy','','critical','security','false','false',NULL,24,1,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,24,'admin@paladincloud.io',now(),null,'ENABLED','rapid7');
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('gcp_vm_r7_resource_scanned','asset_lookup_index','rapid7_vulnerability','','false','false','false','','');
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('gcp_vm_r7_resource_scanned','policyKey','check-asset-scanned-by-source','','false','false','false','','');
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('gcp_vm_r7_resource_scanned','asset_lookup_key','instanceId','','false','false','false','','');
---  start  Rapid7 Gap policy --
+--  end  Rapid7 Gap policy --
 
+UPDATE cf_PolicyTable SET policyDisplayName='Enable Rapid7 Vulnerability Scan' where policyId in ('aws_ec2_r7_resource_scanned','azure_vm_r7_resource_scanned','gcp_vm_r7_resource_scanned');
```

Comment:
@santhosh-challa these updates should be removed from the code, correct? @rnithinpaladin can you add a block on merge if code contains updates, similar what we did with delete statements.